packages/rocketchat-livechat/client/views/sideNav/livechat.js

```diff
@@ -62,3 +85,7 @@ Template.livechat.events({
 		});
 	}
 });
+
+Template.livechat.onCreated(function() {
+ 	this.subscribe('livechat:inquiry');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed spaces and tabs.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2726498730/issues/source?bid=3199634&fileBranchId=3348698#l90)